packages/entity-browser/src/routes/detail/components/ReduxFormFieldAdapter/ReduxFormFieldAdapter.js

```diff
@@ -17,24 +19,25 @@ const ReduxFormFieldAdapter = props => {
     formFieldUtils
   } = props
 
-  return (
-    <div>
-      <FormField
-        id={id}
-        formDefinitionField={formDefinitionField}
-        entityField={entityField}
-        modelField={modelField}
-        value={input.value}
-        onChange={input.onChange}
-        error={error}
-        events={extractEventsFromInput(input)}
-        touched={touched}
-        dirty={dirty}
-        readOnly={submitting}
-        utils={formFieldUtils}
-      />
-    </div>
-  )
+  const readOnly = submitting
```

Comment:
(I personally would neglect this `readOnly` variable and define it directly in the map (`readOnly: submitting`))